src/vm/arm64/asmhelpers.S

```diff
@@ -288,6 +267,49 @@ WRITE_BARRIER_ENTRY JIT_WriteBarrier
     dmb  ST
     str  x15, [x14]
 
+#ifdef WRITE_BARRIER_CHECK
+    // Update GC Shadow Heap
+
+    // need temporary registers. Save them before using.
+    stp  x12, x13, [sp, #-16]!
+
+    // Compute address of shadow heap location:
+    //   pShadow = g_GCShadow + (x14 - g_lowest_address)
+    ldr  x12, =g_lowest_address
```

Comment:
@janvorli OK, I'll take a look tomorrow.  I don't remember writing any code to access global variables.  Presumably arm32 has prototypical usage that I need to follow.

Since arm64 this doesn't work on Android anyway, should we just make that a separate PR